File: SmartReceipts/PDF/TableHeaderRow.m
//
//  TableHeaderRow.m
//  SmartReceipts
//
//  Created by Jaanus Siim on 10/07/15.
//  Copyright (c) 2015 Will Baumann. All rights reserved.
//

#import "TableHeaderRow.h"

@interface TableHeaderRow ()

@property (nonatomic, strong) IBOutlet UIView *bottomSeparatorView;

@end

IB_DESIGNABLE
@implementation TableHeaderRow

- (void)awakeFromNib {
    [super awakeFromNib];

    UIColor *styleColor = [Customization reportPDFStyleColor];
    [self.contentLabel setTextColor:styleColor];
    [self setBackgroundColor:[styleColor colorWithAlphaComponent:0.2]];
    [self.bottomSeparatorView setBackgroundColor:styleColor];
}

@end

File: SmartReceipts/Reports/PDF/TableHeaderRow.swift
//
//  TableHeaderRow.swift
//  SmartReceipts
//
//  Created by Bogdan Evsenev on 01/12/2017.
//  Copyright © 2017 Will Baumann. All rights reserved.
//

import Foundation

@IBDesignable
class TableHeaderRow: TableContentRow {
    @IBOutlet private var bottomSeparatorView: UIView?
    
    override func awakeFromNib() {
        super.awakeFromNib()
        
        let styleColor = AppTheme.reportPDFStyleColor
        contentLabel?.textColor = styleColor
        backgroundColor = styleColor.withAlphaComponent(0.2)
        bottomSeparatorView?.backgroundColor = styleColor
    }
}
